src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+    <variable>
+      <name>CNDC_2</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+        <attribute name="standard_name" value="'sea_water_electrical_conductivity'"/>
+        <attribute name="long_name" value="'electrical conductivity of the water body'"/>
+        <attribute name="units" value="'siemens per metre'"/>
+        <attribute name="valid_min" value="0.f"/>
+        <attribute name="valid_max" value="50000.f"/>
+        <attribute name="_FillValue" value="-9999.f"/>
+        <attribute name="ancillary_variables" value="'CNDC_2_quality_control'"/>
+      </attributes>
+    </variable>
+    <variable>
+      <name>CNDC_2_quality_control</name>
+      <encoder>char</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+        <attribute name="standard_name" value="'sea_water_electrical_conductivity status_flag'"/>
+        <attribute name="long_name" value="'quality flag for sea_water_electrical_conductivity'"/>
+        <attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+    <variable>
+      <name>CNDC_3</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+        <attribute name="standard_name" value="'sea_water_electrical_conductivity'"/>
+        <attribute name="long_name" value="'electrical conductivity of the water body'"/>
```

Comment:
Could we have more info about what distinguishes CNDC from CNDC_2, CNDC_3... ? Variable attribute 'comment' could be used.